lib/manager/helm-values/update.ts

```diff
@@ -0,0 +1,82 @@
+import YAWN from 'yawn-yaml/cjs';
+import { logger } from '../../logger';
+import { Upgrade } from '../common';
+import { matchesHelmValuesDockerHeuristic } from './util';
+
+function shouldUpdate(
+  parentKey: string,
+  data: any,
+  depName: string,
+  currentValue: string
+): boolean {
+  return (
+    matchesHelmValuesDockerHeuristic(parentKey, data) &&
+    data.repository === depName &&
+    data.tag === currentValue
+  );
+}
+
+/**
+ * Recursive function that walks the yaml strucuture
+ * and updates the first match of an 'image' key it finds,
+ * if it adheres to the supported structure.
+ *
+ * @param parsedContent The part of the yaml tree we should look at.
+ * @param depName The name of the dependency that should be updated.
+ * @param currentValue The current version that should be updated.
+ * @param newValue The update version that should be set instead of currentValue.
+ */
+function updateDoc(
+  parsedContent: object,
+  depName: string,
+  currentValue: string,
+  newValue: string
+): boolean {
+  for (const key in parsedContent) {
+    if (Object.prototype.hasOwnProperty.call(parsedContent, key)) {
```

Comment:
```suggestion
    {
```
check can be removed with above for change